### PR TITLE
Thank you page

### PR DESCRIPTION
**Pull request by gunnaraasen**

### CODE REVIEW

File: content/support/dev-support-purchased.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
++++
+title = "Developer Support Purchased"
++++
+
+# Thank you!
+
+You have successfully signed up for the Developer Support Plan.
+
+Emails sent to [support@influxdb.com](mailto:support@influxdb.com) originating from your email will be addressed within six business hours. Business hours are 8am to 8pm PST.
```
Comment (beckettsean) [inline content/support/dev-support-purchased.md]: 8 a.m. to 8 p.m. Eastern Time

File: content/support/dev-support-purchased.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Receipts will automatically be emailed every time you are charged.
```
Comment (beckettsean) [inline content/support/dev-support-purchased.md]: Add: "Your subscription will renew monthly until cancelled.

File: content/support/dev-support-purchased.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Please [email support](mailto:support@influxdb.com) for all account administration requests, such as,
+
+- requesting receipts be sent to a different email
+- changing the email address that will receive support
+- changing your subscription
```
Comment (beckettsean) [inline content/support/dev-support-purchased.md]: changing or canceling your subscription

**Comment by beckettsean** [comment]
A few comments, otherwise +1

**Comment by gunnaraasen** [comment]
Made the changes and put them up on new.

**Pull request merged**